app/src/test/java/org/simple/clinic/bp/entry/BloodPressureValidationMockDateValidatorTest.kt

```diff
@@ -186,15 +179,14 @@ class BloodPressureValidationMockDateValidatorTest {
   private fun instantiateFixture(openAs: OpenAs) {
     val effectHandler = BloodPressureEntryEffectHandler(
         ui = ui,
-        userSession = userSession,
-        facilityRepository = facilityRepository,
         patientRepository = patientRepository,
         bloodPressureRepository = bloodPressureRepository,
         appointmentsRepository = appointmentRepository,
         userClock = testUserClock,
-        schedulersProvider = TrampolineSchedulersProvider(),
-        uuidGenerator = FakeUuidGenerator.fixed(UUID.fromString("7283abf4-b718-4379-b101-46f011b5536b"))
-    ).build()
+        schedulersProvider = TestSchedulersProvider.trampoline(),
+        uuidGenerator = FakeUuidGenerator.fixed(UUID.fromString("7283abf4-b718-4379-b101-46f011b5536b")),
+        currentUser = { user }
+    ) { facility }.build()
```

Comment:
Can we move this inside the constructor, please?

Reply:
Yup, moving it.